source/ed/osm2ed.cpp

```diff
@@ -754,6 +754,18 @@ void OSMAdminRelation::build_polygon(OSMCache& cache) {
             }
             break;
         }
+
+        if (boost::geometry::area(tmp_polygon) < std::numeric_limits<double>::epsilon()) {
```

Comment:
Duplicated code spotted ! Would be nice to have a common function that does this.